controllers/LoginController.php

```diff
@@ -53,7 +53,8 @@ public function Auth()
       $options['iitpConnect_user_state'] = 'logged_in';
 
       $session->set($options);
-      $result = array('response' => 'success', 'text' => User::$username, 'message' => 'login successfull');
+      User::isOnline();
```

Comment:
Handel error these of return value is false for true.

Reply:
What of user get logged in but due to some issue `isOnline` not able to log in. So this is not good.
So send userid as a param to `isOnline` function.